src/EditorFeatures/TestUtilities/Remote/InProcRemostHostClient.cs

```diff
@@ -26,8 +26,8 @@ public static async Task<RemoteHostClient> CreateAsync(Workspace workspace, bool
         {
             var inprocServices = new InProcRemoteServices(runCacheCleanup);
 
-            var remoteHostStream = await inprocServices.RequestServiceAsync(WellKnownRemoteHostServices.RemoteHostService, cancellationToken).ConfigureAwait(false);
             var remotableDataRpc = new RemotableDataJsonRpc(workspace, inprocServices.Logger, await inprocServices.RequestServiceAsync(WellKnownServiceHubServices.SnapshotService, cancellationToken).ConfigureAwait(false));
+            var remoteHostStream = await inprocServices.RequestServiceAsync(WellKnownRemoteHostServices.RemoteHostService, cancellationToken).ConfigureAwait(false);
```

Comment:
If this was moved for a specific reason, what is the reason? (and maybe add a comment explaining)

Reply:
@heejaechang What's the right explanation here? :smile:

Reply:
remoteableDataRpc call will set up IExperimentService in OOP. that will make sure all other calls to succeed when asking IExperimentService.

Reply:
Added my best paraphrasing of @heejaechang -- hopefully it's still correct. :smile: